app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
     pass
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list) -> float:
```

Comment:
The Car annotation cannot be specified here, as a floating-point value is returned

Reply:
You should specify the type annotation for argument `car_list` from indicating that you expect to obtain a list to expecting obtaining a list of `Car` class instances. It does not relate to the return type annotation. Besides that, it would be beneficial to change variable naming from `car_list` to `cars` to not indicate the type of the variable in its name without necessity.